idx/idx.go

```diff
@@ -18,10 +18,27 @@ var (
 type Node struct {
 	Path        string
 	Leaf        bool
-	Defs        []schema.MetricDefinition
+	Defs        []Archive
 	HasChildren bool
 }
 
+type Archive struct {
```

Comment:
it describes an archive though. I first wanted to call this ArchiveMeta but felt that was a bit too much.